### ISSUE TITLE
oc_auth

### ISSUE THREAD

**Issue by downrightheight** [OpenCollar/opencollar#606]
These lines in oc_auth look wrong, how can a variable have 2 values without being reassigned?
lines 598 and 599

```
} else if (sCommand == "runaway") {
    if (sCommand == "on") {
```

**Comment by GarvinTwine** [comment]
seems weird indeed...
599 should have been if (sAction == "on") {
and the same a few lines later, thanks for this catch.

**Comment by GarvinTwine** [comment]
fixed in #608
